File: dataset_reduzido/codenet/Java/Code/s890213274.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A1 = sc.nextInt();
        int A2 = sc.nextInt();
        int A3 = sc.nextInt();
        
        if ((A1+A2+A3) >= 22) {
            System.out.println("bust");
        } else {
            System.out.println("win");
        }
    
    }
}

File: output/Qwen/codenet/Java/Python/s890213274.py
#!/usr/bin/env python3

def main():
    sc = Scanner()
    A1 = sc.nextInt()
    A2 = sc.nextInt()
    A3 = sc.nextInt()
    
    if (A1 + A2 + A3) >= 22:
        print("bust")
    else:
        print("win")

if __name__ == "__main__":
    main()

class Scanner:
    def __init__(self):
        self.input = input().split()
        self.index = 0
    
    def nextInt(self):
        result = int(self.input[self.index])
        self.index += 1
        return result